modules/configure-acs-kube-security-platform.adoc

```diff
@@ -0,0 +1,52 @@
+[id="configure-acs-kube-security-platform"]
+= Configuring the {product-title} platform
+:context: configure-acs-kube-security-platform
+
+You can create a new integration in the {product-title} platform by using the email plugin notifier.
+
+.Procedure
+
+. Navigate to *Platform Configuration* -> *Integrations*.
+. Under the *Plugins* section, select *Email*.
+. Select the *New Integration* icon.
+. Enter a name for your email integration in the *Integration Name*.
+. For *Email Server*, enter the address of your email server.
+. Enter the `_Username_` and `_Password_` to use for authentication.
+. Optional: For *FROM*, specify the `_email FROM header_`.
+. For *SENDER*, specify the `_email SENDER header_`.
+. Enter the email address which will receive the notifications in the *Default Recipient* box or enter an annotation key for *Annotation key for recipient*.
```

Comment:
This doesn't tell me what the annotation key for recipient is.

Reply:
Hi @gaurav-nelson I will take a look at the docs to see if I can find the annotation key for the recipient...I might need your expertise here though. Thanks.

Reply:
@gaurav-nelson this is the last update that I need to make before SME review. If I don't get more information about the annotation key, I will get with you on Tuesday. Thanks.